projects_info.csv

```diff
@@ -10,6 +10,7 @@ Y | qgis | 3.22.7 | Y | Y | Desktop GIS | | | qgis | https://www.qgis.org/en/sit
 #   OSGeo Community | | | | | | | | | | | |
 #   Others  | | | | | | | | | | | |
 Y | openjump | 1.15.1 | Y | Y | Desktop GIS | | | openjump_gis | http://www.openjump.org | | OpenJUMP GIS | Y
+Y | qfield | 2.7.6 | Y | Y | Mobile GIS | | | qfield | https://qfield.org/ | | QField | Y
```

Comment:
Do we have the catagory Mobile GIS? Or are we going to add it as new section on the content page?
https://osgeo.github.io/OSGeoLive-doc/en/overview/overview.html

Reply:
I guess that's a question for someone else to answer? I am just following guidance here :-)

Reply:
We should probably not change the structure that late in the process

Reply:
```suggestion
Y | qfield | 2.7.6 | Y | Y | Mobile GIS | | | qfield | https://qfield.org/ | | QField | N
```